ai_user/settings.py

```diff
@@ -49,15 +52,14 @@ async def config(self, ctx: commands.Context):
     @ai_user.command()
     @checks.is_owner()
     async def scan_images(self, ctx: commands.Context):
-        """ Toggle image scanning (see README.md)"""
+        """ Toggle image scanning (see README.md) """
         try:
             importlib.import_module("pytesseract")
             importlib.import_module("torch")
             importlib.import_module("transformers")
             value = not (await self.config.guild(ctx.guild).scan_images())
             await self.config.guild(ctx.guild).scan_images.set(value)
-            embed = discord.Embed(
-                title="⚠️ WILL CAUSE HEAVY CPU LOAD ⚠️")
+            embed = discord.Embed(title="⚠ WILL CAUSE HEAVY CPU LOAD ⚠", color=await ctx.embed_color())
```

Comment:
Is something automatically converting the⚠️ to a more text (?)⚠️? Or is there a reason for doing this?

Reply:
Yours seemed to have some invisible characters in it and I thought that was weird